acme/acme/crypto_util.py

```diff
@@ -203,7 +203,7 @@ def gen_ss_cert(key, domains, not_before=None,
     """
     assert domains, "Must provide one or more hostnames for the cert."
     cert = OpenSSL.crypto.X509()
-    cert.set_serial_number(1337)
+    cert.set_serial_number(OpenSSL.rand.bytes(16))
```

Comment:
Unfortunately, this produces

`TypeError: serial must be an integer`

`OpenSSL.rand.bytes()` returns a `str` (probably `bytes` in Python 3), but `set_serial_number` wants an `int`. 

How about

`cert.set_serial_number(int(OpenSSL.rand.bytes(16).encode("hex"), 16))`

?